packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -612,12 +618,13 @@ class DragTarget<T extends Object> extends StatefulWidget {
     super.key,
     required this.builder,
     this.onWillAccept,
+    this.onWillAcceptWithDetails,
     this.onAccept,
     this.onAcceptWithDetails,
     this.onLeave,
     this.onMove,
     this.hitTestBehavior = HitTestBehavior.translucent,
-  });
+  }): assert(onWillAccept == null || onWillAcceptWithDetails == null, 'Both onWillAccept and onWillAcceptWithDetails must not be provided.');
```

Comment:
Formatting nit:

```suggestion
  }) : assert(onWillAccept == null || onWillAcceptWithDetails == null, 'Both onWillAccept and onWillAcceptWithDetails must not be provided.');
```

Reply:
Also nit: I think this is slightly more clear: "Don't pass both onWillAccept and onWillAcceptWithDetails."